src/CalcViewModel/Common/CopyPasteManager.h

```diff
@@ -13,27 +13,26 @@ namespace CalculatorUnitTests
 
 namespace CalculatorApp
 {
-
-#define QwordType 1
-#define DwordType 2
-#define WordType 3
-#define ByteType 4
-#define HexBase 5
-#define DecBase 6
-#define OctBase 7
-#define BinBase 8
+    inline constexpr auto QwordType = 1;
+    inline constexpr auto DwordType = 2;
+    inline constexpr auto WordType = 3;
+    inline constexpr auto ByteType = 4;
+    inline constexpr auto HexBase = 5;
+    inline constexpr auto DecBase = 6;
+    inline constexpr auto OctBase = 7;
+    inline constexpr auto BinBase = 8;
 
     class CopyPasteManager
     {
     public:
         static void CopyToClipboard(Platform::String^ stringToCopy);
         static concurrency::task<Platform::String^> GetStringToPaste(CalculatorApp::Common::ViewMode mode, CalculatorApp::Common::CategoryGroupType modeType, int programmerNumberBase = -1, int bitLengthType = -1);
-        static bool HasStringToPaste()
+        inline static bool HasStringToPaste()
```

Comment:
```suggestion
        static bool HasStringToPaste()
```
According to [`inline`](https://en.cppreference.com/w/cpp/language/inline):
> A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function.